src/components/error-message/default/index.njk

```diff
@@ -2,8 +2,34 @@
 layout: layout-example.njk
 ---
 
-{% from "error-message/macro.njk" import govukErrorMessage %}
+{% from "date-input/macro.njk" import govukDateInput %}
 
-{{ govukErrorMessage({
-  text: "Error message goes here"
-}) }}
+{{ govukDateInput({
+  fieldset: {
+    legend: {
+      text: 'What is your date of birth?',
+      isPageHeading: true,
+      classes: 'govuk-fieldset__legend--xl'
+    } 
+  },
+  hint: {
+    text: 'For example, 31 3 1980'
+  },
+  errorMessage: {
+    text: "Date of birth must be in the past"
+  },
+  id: 'dob',
+  name: 'dob',
+  items: [
+    {
+      name: 'day'
```

Comment:
Should these include values like in `date-input/default/index.njk`?

Reply:
Updated to add the values